example/fail/sub/sub.go

```diff
@@ -0,0 +1,9 @@
+package sub
+
+func ok() {
+	_ = "ok"
+}
+
+func CoveredFromRoot() {
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function CoveredFromRoot should have comment or be unexported